### DISCUSSION TITLE
Total noob

### DISCUSSION THREAD

**Discussion by leelaBQ** [Dineshkarthik/telegram_media_downloader#341] [Q&A] [answered]
Hello there, im sorry for bothering you friends, but I can´t make this work, I don´t know what things do I have to install/configure before being able to use the following commands:
$ git clone https://github.com/Dineshkarthik/telegram_media_downloader.git
$ cd telegram_media_downloader
$ pip3 install -r requirements.txt
I have python3, but I dont know if im opening the right terminal and also it says that it doesnt recognizes the "$", so probably I'm missing stuff, what do I need to install to be able to run this? :D Thanks for the help, I hope I'm not bothering.

So... I have a sugestion along with my questions, probably this github would benefit with a more noob friendly readme :D <3

**Comment by omshivaprakash** [comment, marked as answer]
$ is your shell - type the command. Next to that.

On Sun, 11 Sep 2022 at 3:42 AM, leelaBQ ***@***.***> wrote:

> Hello there, im sorry for bothering you friends, but I can´t make this
> work, I don´t know what things do I have to install/configure before being
> able to use the following commands:
> $ git clone https://github.com/Dineshkarthik/telegram_media_downloader.git
> $ cd telegram_media_downloader
> $ pip3 install -r requirements.txt
> I have python3, but I dont know if im opening the right terminal and also
> it says that it doesnt recognizes the "$", so probably I'm missing stuff,
> what do I need to install to be able to run this? :D Thanks for the help, I
> hope I'm not bothering.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/Dineshkarthik/telegram_media_downloader/issues/323>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAA4EEFK3C3OHZNCAC4D5F3V5UBVHANCNFSM6AAAAAAQJQFMMA>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>
-- 
Sent from Gmail Mobile
